packages/components/src/DraftConfigurator/OptionGroup/index.js

```diff
@@ -66,13 +66,19 @@ const OptionGroup = (props) => {
 
   return (
     <>
-      {props.options.map((name) => {
+      {props.options.map((nameOrGroup) => {
         let output = []
```

Comment:
You're a brave man for wading into this mess ;)

I'm putting this comment on a code line you did not touch, because it is something that my older code is very guilty of, but: I find code is easier to read when you only use `let` for variables that need to change. And `const` for everything else.

So I'm going to add comments each time I see a `let` that could/should be a `const` knowing full well that I have (at least historically) not been very good at that either.

So see this more as a generic code style feedback that I want to share with you, and not as criticism

Reply:
Sounds great. I think I actually improved a bit on this since I sent this PR. My latest PR is full of `const`!